apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -33,7 +37,36 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
     await this.worker.run<CreateHnippNotificationDto>(
       async (message, job): Promise<void> => {
         const messageId = job.id
-        this.logger.info('Message received by worker ... ...', { messageId })
+        this.logger.info('Message received by worker 1.1', { messageId })
+
+        const notification = { messageId, ...message }
+        const messageIdExists = await this.notificationModel.count({
+          where: { messageId },
+        })
+
+        if (messageIdExists > 0) {
+          // messageId exists do nothing
+          this.logger.info('notification already exists 1.1', {
```

Comment:
Wondering if we should downgrade this and the log in L58 to debug as it could be a lot of logs in production.